### ISSUE TITLE
cutom dbs won't download

### ISSUE THREAD

**Issue by dmb-ts** [extremeshok/clamav-unofficial-sigs#117]
Hi.

I just switched to the new version (my first switch to the new configuration way - with master.conf und user.conf) and I can't add custom dbs.

When I run clamav-unofficial-sigs.sh I get:
```
2016-05-23 11:08:45 URL:https://raw.githubusercontent.com/wmetcalf/clam-punch/master/miscreantpunch099.ldb [53838/53838] -> "/var/lib/clamav-unofficial-sigs/dbs-add/miscreantpunch099.ldb" [1]
Failed connection to raw.githubusercontent.com - SKIPPED miscreantpunch099.ldb update
Testing updated database file: /var/lib/clamav-unofficial-sigs/dbs-add/miscreantpunch099.ldb
Clamscan reports User Added /var/lib/clamav-unofficial-sigs/dbs-add/miscreantpunch099.ldb database integrity tested BAD
Removed invalid database: /var/lib/clamav-unofficial-sigs/dbs-add//var/lib/clamav-unofficial-sigs/dbs-add/miscreantpunch099.ldb
```

But a `wget https://raw.githubusercontent.com/wmetcalf/clam-punch/master/miscreantpunch099.ldb` works fine and downloads the correct ldb file. And clamav can load this file without problem.

My master.conf contains the following lines:
```
additional_dbs="
#   rsync://192.168.1.50/new-db/sigs.hdb
#   rsync://rsync.example.com/all-dbs/
#   ftp://ftp.example.net/pub/sigs.ndb
#   http://www.example.org/sigs.ldb
https://raw.githubusercontent.com/wmetcalf/clam-punch/master/miscreantpunch099.ldb
" #END ADDITIONAL DATABASES
```
Is it broken or did I do something wrong?

PS:
everything else works fine

**Comment by extremeshok** [comment]
will test shortly.

**Comment by extremeshok** [comment]
fixed in latest dev, will push an update shortly

**Comment by extremeshok** [comment]
fixed in 5.3.2

**Comment by extremeshok** [comment]
Note: this database has an extremely hight FP rating.
Mainly caused by : MiscreantPunch.MacroHeurAOOBR.1.UNOFFICIAL and .xls attachments
